### DISCUSSION TITLE
04_judi_leading_edge_tutorial.ipynb

### DISCUSSION THREAD

**Discussion by venividivici666** [slimgroup/JUDI.jl#144] [Q&A]
SLIM_optim not defined

**Comment by huangzilin** [comment]
您好！已收到您的邮件！

**Reply by mloubout** [reply]
This tutorial requires the release associated with it to have the same version as the one that was used at the time:

https://github.com/slimgroup/JUDI.jl/releases/tag/untagged-477a525d48cc92e0050b
